www/js/app.js

```diff
@@ -768,10 +833,16 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                 $('#articleListHeaderMessage').empty();
                 $('#articleListWithHeader').hide();
                 $("#prefix").val("");
-                iframeArticleContent.onload = function () {
+                iframeArticleContent.onload = function() {
                     // The content is fully loaded by the browser : we can hide the spinner
-                    iframeArticleContent.onload = function () {};
```

Comment:
I don't remember why this line was necessary in the past. Maybe it's not necessary any more, I suppose you checked?

Reply:
It was necessary to prevent the onload function happening every time an article is loaded in the Service Worker. However, for this code to work, we now need that function to run. It has no negative effects, and we are not altering the html of the document loaded by the SW, but we are attaching an event to the body tag of that document, once it has completed loading. I found no other way of monitoring a dragover on the iframe (and I tried quite a few ways), attaching it to the contentWindow, attaching it to the contentDocument, attaching it to the iframe itself, none worked.

Reply:
All right, thanks for the explanation